### DISCUSSION TITLE
chooseVisitors

### DISCUSSION THREAD

**Discussion by SoltauFintel** [spotbugs/spotbugs-gradle-plugin#687] [Q&A]
hi

I want to migrate from FindBugs 3 to SpotBugs 4.8.0. How can I migrate this Ant setting to Gradle:

chooseVisitors="+CallToUnsupportedMethod,+CheckExpectedWarnings,+CovariantArrayAssignment,+InefficientIndexOf,+InefficientInitializationInsideLoop,+InefficientMemberAccess,+InefficientToArray,+PublicSemaphores,+UselessSubclassMethod"

**Comment by SoltauFintel** [comment]
see issue #690

**Comment by KengoTODA** [comment]
try `visitors` option.

https://github.com/spotbugs/spotbugs-gradle-plugin/tree/c5b6c583839bb1bd127deb9e30f8d35c47a42f33#configure-spotbugs-plugin

**Reply by SoltauFintel** [reply]
Thanks for the answer. This is just a workaround, but not a good solution. Instead of 6 chooseVisitors entries you have to enter 100 visitors entries. (see the issue)
